packages/kit/src/packaging/index.js

```diff
@@ -326,8 +331,7 @@ async function try_load_svelte2tsx() {
 			return await import('svelte2tsx');
 		} catch (e) {
 			throw new Error(
-				'You need to install svelte2tsx and typescript if you want to generate type definitions\n' +
-					e
+				'You need svelte2tsx and typescript if you want to generate type definitions. Install it through your package manager, or disable generation which are highly discouraged. See https://kit.svelte.dev/docs#packaging'
```

Comment:
```suggestion
				'You need svelte2tsx and typescript if you want to generate type definitions. Install it through your package manager, or disable generation which is highly discouraged. See https://kit.svelte.dev/docs#packaging'
```

Should this be an "is"?

Reply:
Yeah, I think I mixed up the wording somewhere in between 😄